### PR TITLE
attachment specs

### PR DESCRIPTION
**Pull request by kaeline**
https://github.com/Metatavu/lipsanen-project-management-spec/issues/71

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -536,6 +536,183 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /v1/projects/{project}/attachments:
```
Comment (Daniilmass) [inline swagger.yaml]: Should we consider passing the project id as an optional query parameter :D Could we end up in the same situation as with i.e. project/tasks? Just thoughts

File: swagger.yaml [hunk 2]
```diff
@@ -2203,6 +2380,29 @@ components:
         metadata:
           $ref: '#/components/schemas/Metadata'
 
+    Attachment:
+      type: object
+      description: Attachment object
+      required:
+        - projectId
+        - url
+      properties:
```
Comment (Daniilmass) [inline swagger.yaml:2369]: How about "name" and "type"?

**Review by Daniilmass — APPROVED** [review]

File: swagger.yaml (hunk 1 above)
Comment (kaeline) [reply swagger.yaml]: If you think we might end in situation where we need listing all attachments, then ofc, I will change the path :)

File: swagger.yaml (hunk 2 above)
Comment (kaeline) [reply swagger.yaml:2369]: name and type are added as required properties

**Review by lauriLukkarinen — APPROVED** [review]

**Review by lauriLukkarinen — APPROVED** [review]

**Pull request merged**